recipes/ms-gsl/all/conanfile.py

```diff
@@ -43,8 +43,13 @@ def _compilers_minimum_version(self):
             "apple-clang": "3.4",
         }
 
+    def package_id(self):
```

Comment:
and config_options is called before package_id also, this is why I said to move validate() ;)

Reply:
Done.